### DISCUSSION TITLE
Change the X and Y axis range

### DISCUSSION THREAD

**Discussion by joselivera** [Kanaries/pygwalker#139] [Q&A]
How I can change the X and Y axis? 
I know that I can change the values that are show by applying a filter into the variable of the label, but the axis tick scale still the same. Is there a way to change this range?

In the image bellow, I applied a filter to the range of duration, but the axis still the same.
![image](https://github.com/Kanaries/pygwalker/assets/62005170/f2d51e24-18b1-454b-9f0a-06145ea294ee)

**Comment by pankajAppPerfect** [comment]
> How I can change the X and Y axis? I know that I can change the values that are show by applying a filter into the variable of the label, but the axis tick scale still the same. Is there a way to change this range?
> 
> In the image bellow, I applied a filter to the range of duration, but the axis still the same. ![image](https://private-user-images.githubusercontent.com/62005170/248267448-f2d51e24-18b1-454b-9f0a-06145ea294ee.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyNjMyNjMsIm5iZiI6MTcwNDI2Mjk2MywicGF0aCI6Ii82MjAwNTE3MC8yNDgyNjc0NDgtZjJkNTFlMjQtMThiMS00NTRiLTlmMGEtMDYxNDVlYTI5NGVlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAzVDA2MjI0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyZGJjZjdhYTAyZmE4NjFkYWZlNjA5ZjAxYWNiYjAwY2E2OGM4YjQwOGY3YjcxZjdmNjk1MWUxMTJlMDRhNmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.jACk9yNNAkgEIldETCQ57ZnTUAcQzZuaEdco9H_xd28)

Hey!! have you got your answer? I am facing same issue

**Comment by longxiaofei** [comment]
1. click config

![image](https://github.com/Kanaries/pygwalker/assets/28337703/31da7c67-2104-4a26-b471-dcf0c0186288)

2. cancel Zero Scale

![image](https://github.com/Kanaries/pygwalker/assets/28337703/ec4d782b-bca8-4944-b026-1277f8582a7d)

3. result

![image](https://github.com/Kanaries/pygwalker/assets/28337703/b9c43810-c9e5-4097-a520-ec06ba5378ca)

**Reply by pankajAppPerfect** [reply]
yeah, it will set axis range according to datapoints but can we set a custom value of ranges?

**Reply by longxiaofei** [reply]
I added it to the todo list.

**Reply by pankajAppPerfect** [reply]
Thanks! looking forward to have this functionality from GUI
